packages/phalanx/src/Phalanx_CreateDeviceEvaluator.hpp

```diff
@@ -24,7 +24,6 @@ namespace PHX {
                          {
                            new (e) DeviceEvaluatorType(evaluator_to_clone_on_device);
                          });
-    ExecutionSpace().fence();
```

Comment:
@MicheldeMessieres - this is experimental code that would not be caught by standard testing. It needs some special configure flags to enable along with relocatable device code. Can we hold off on removing until I can do some more extensive testing?

Reply:
@rppawlo Yes - sorry - I 'tested' for this to see if it was used, and then all tests passed and forgot what I was doing ... we can just treat this as a 'fence review' to tag internal fences. Fine with me if you want to just close this for now.